moveit_experimental/jog_arm/include/jog_arm/low_pass_filter.h

```diff
@@ -44,8 +44,7 @@ namespace jog_arm
 {
 /**
  * Class LowPassFilter - Filter a signal to soften jerks.
- * This is a second-order Butterworth low-pass filter.
- * See https://ccrma.stanford.edu/~jos/filters/Example_Second_Order_Butterworth_Lowpass.html
+ * This is a first-order Butterworth low-pass filter.
```

Comment:
Could you add a TODO suggested we consolidate our filter to use 
https://github.com/ros/filters

As @gavanderhoorn suggested. I also opened this related issue: https://github.com/ros/filters/issues/28